bootstrap_cfn/stacks/ec2.json

```diff
@@ -8,11 +8,26 @@
         "SecurityGroupIngress": []
     }
   },
+  "SaltSG": {
+    "Type": "AWS::EC2::SecurityGroup",
+    "Properties": {
+        "VpcId" : { "Ref" : "VPC" },
+        "GroupDescription": "Salt Comms Security Group",
+        "SecurityGroupIngress": [
+                                    {
+                                        "IpProtocol": "tcp",
+                                        "FromPort": 4505,
+                                        "ToPort": 4506,
```

Comment:
Think it might be okay as is.

See [here](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-security-group-ingress.html)

What I intend it to be is open ports 4505 and 4506 inbound to anyone in BaseHostSG